### PR TITLE
Localize content/en/docs/reference/glossary/annotation.md into Hindi

### PR DESCRIPTION
**Pull request by taniaduggal**
Fixes #35600

### CODE REVIEW

File: content/hi/docs/reference/glossary/cidr.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: सीआईडीआर(CIDR)
+id: cidr
+date: 2019-11-12
```
Comment (bishal7679) [inline content/hi/docs/reference/glossary/cidr.md:4]: @rolfedh should we update the date as well with the current date?

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

**Review by bishal7679 — COMMENTED** [review]
@rolfedh Should we update the date 

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Comment by rolfedh** [comment]
@taniaduggal did you mean to close this PR?

File: content/hi/docs/reference/glossary/cidr.md (hunk 1 above)
Comment (rolfedh) [reply content/hi/docs/reference/glossary/cidr.md:4]: @sftim For localizations, do we preserve or reset the date?

Comment (sftim) [reply content/hi/docs/reference/glossary/cidr.md:4]: No opinion. We don't use that value anywhere when rendering the site. Individual localizations can pick what approach they want.

**Pull request closed without merging**